.github/workflows/pr.yml

```diff
@@ -15,13 +15,6 @@ env:
   NUGET_PACKAGES: ${{ github.workspace }}/.nuget/packages
 
 jobs:
-  docs:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - name: Build documentation
-        uses: elastic/docs-builder@main
```

Comment:
Is it possible to use the local action?

```yaml
steps:
  - uses: actions/checkout@v4
  - uses: .
```

This would the use the action of the checked-out ref.

Reply:
it is not, our GitHub action uses a prebuilt container image that gets published from `main`. 

This ensures our Github Action is *ready to run* and no-one needs to wait for it to build the artifacts first. 

This ensures our GitHub actions runs in seconds for everyone else 😸